pkg/util/shell/shell.go

```diff
@@ -90,7 +90,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 		prefix = "SET "
 		suffix = "\n"
 		if pathVar {
-			suffix = ";%PATH%" + suffix
+			suffix = ";%PATH%;" + suffix
```

Comment:
it would still be OK... as it places a `;` at the end... this seems unnecessary... and if the 'user' does this himself, it is his repsonsibility